fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -403,6 +404,24 @@ public void testGetNonRDFSourceDescription() throws IOException {
         }
     }
 
+    /**
+     * Test that a 406 gets returned in the event of an invalid or unsupported
+     * format being requested.
+     */
+    @Test
+    public void testGetRDFSourceWrongAccept() throws IOException {
+        final String id = getRandomUniqueId();
+        createObjectAndClose(id);
+
+        final HttpGet get = new HttpGet(serverAddress + id);
+        get.addHeader("Accept", "application/turtle");
+
+        try (final CloseableHttpResponse response = execute(get)) {
```

Comment:
I think there's a `getStatus` method in the superclass that does the closing for you.